### DISCUSSION TITLE
Inconsistent dose and density data files

### DISCUSSION THREAD

**Discussion by Ahsong9527** [nrc-cnrc/EGSnrc#962] [Questions] [answered]
Hello researchers !
I follow the getting-started(7.5 Calculate dose with DOSXYZnrc),and then found
![image](https://user-images.githubusercontent.com/116752446/218622624-60fb6877-2336-4c31-9c53-b62916c36807.png)
This is my .egsphant and .3ddose files [files](https://github.com/Ahsong9527/question.git)

**Comment by rtownson** [comment, marked as answer]
Your 3ddose file is 14x14x5 voxels, but the phantom is 88x88x50, so your simulation was not set to use the egsphant file. Make sure to select "phantom created from CT data", and select your egsphant file, and then run the simulation again.

**Comment by Ahsong9527** [comment]
![image](https://user-images.githubusercontent.com/116752446/219253295-7553ff73-02cc-4026-8762-a928a5534f3e.png)
Sorry to bother you again,I don't know what happen.As you say,I select my egsphant file and run again,make sure the ct data is 80x80x50 but  it still not work.I don't know what the bug mean.

**Reply by rtownson** [reply]
That error means that your system is missing some fonts that are normally used by the GUI. Honestly it is a pain to fix, and I recommend you just switch to the better tool, VICTORIA:
http://web.uvic.ca/~bazalova/VICTORIA/

See more options here:
https://github.com/nrc-cnrc/EGSnrc/wiki/Visualizing-or-parsing-3ddose-files

**Comment by Ahsong9527** [comment]
![image](https://user-images.githubusercontent.com/116752446/219539102-ae931077-86d0-408c-80a5-23cb1ce7b0e6.png)
thank so much,l make it as you say!
